src/modelconfig.cpp

```diff
@@ -406,6 +414,26 @@ Status ModelConfig::parseNode(const rapidjson::Value& v) {
         modelVersionPolicy = ModelVersionPolicy::getDefaultVersionPolicy();
     }
 
+    SPDLOG_DEBUG("Specified model parameters:");
+    SPDLOG_DEBUG("model_basepath: {}", getBasePath());
+    SPDLOG_DEBUG("model_name: {}", getName());
+    SPDLOG_DEBUG("batch_size: {}", getBatchSize());
+    if (isShapeAnonymous()) {
+        SPDLOG_DEBUG("shape: {}", std::string(getShapes().begin()->second));
+    } else {
+        SPDLOG_DEBUG("shape:");
+        for (auto& shapeMap : getShapes()) {
+            SPDLOG_DEBUG("  {}: {}", shapeMap.first, std::string(shapeMap.second));
+        }
+    }
+    SPDLOG_DEBUG("model_version_policy: {}", std::string(*getModelVersionPolicy()));
+    SPDLOG_DEBUG("nireq: {}", getNireq());
+    SPDLOG_DEBUG("target_device: {}", getTargetDevice());
+    SPDLOG_DEBUG("plugin_config:");
+    for (auto& pluginMap : getPluginConfig()) {
```

Comment:
Minor:
Instead of using ".first" & ".second" what you can do is
for (auto& [pluginParameter, pluginValue] : getPluginConfig())